### DISCUSSION TITLE
Bio field not support emoji 🙃

### DISCUSSION THREAD

**Discussion by M97Chahboun** [community/community#69314] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![Screenshot 2023-10-05 at 13 29 35](https://github.com/community/community/assets/69054810/ed5b10b7-3ef4-4506-917b-8112b3468d63)

I Added 🤍 but I think emoji not supported

**Comment by himanshu-03** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> ![Screenshot 2023-10-05 at 13 29 35](https://user-images.githubusercontent.com/69054810/272903093-ed5b10b7-3ef4-4506-917b-8112b3468d63.png)
> 
> I Added 🤍 but I think emoji not supported

Hey @M97Chahboun !
I tried adding the same emoji in my profile bio and it is working perfectly fine.

![image](https://github.com/community/community/assets/97957777/f81aac9b-624f-4af1-b1ed-d9f2fd82573d)

**Reply by M97Chahboun** [reply]
Thanks, the issue caused by bio length limit

**Comment by himanshu-03** [comment]
The emoji might not be supported on your device
Try accessing the emoji from your laptop or any other system.
This will surely display the emoji then

**Reply by M97Chahboun** [reply]
Thanks, the issue caused by bio length limit
when I used emoji on (2 remaining) the emoji not displayed as expected
